README-c++11.md

```diff
@@ -0,0 +1,23 @@
+## Availability of Colvars features on different C++ versions
+
+The majority of the Colvars module can be built with all major versions of the C++ language.  A few recent features are an exception, and require an executable built with C++11.
+
+Because NAMD, LAMMPS or VMD do not support C++11 on all platforms yet, those features may be disabled.  This page points to the relevant information to identifying and potentially solving this issue.
+
+
+### Specific Colvars features that require C++11
+
+Currently, the `gspath` and `gzpath` collective variables are only available when the code as been built with C++11 standard or higher.
```

Comment:
`gspathCV` and `gzpathCV` also require C++11. Maybe it's better to list them together.

Reply:
Done.